packages/react-storage/src/components/StorageBrowser/Views/Controls/Message.tsx

```diff
@@ -88,20 +94,22 @@ interface MessageControlProps {
 interface _MessageControl<
   T extends StorageBrowserElements = StorageBrowserElements,
 > extends Pick<T, 'Icon' | 'View'> {
-  ({ variant }: MessageControlProps): React.JSX.Element;
+  ({ variant }: MessageControlProps): React.JSX.Element | null;
 }
 
 export interface MessageControl<
   T extends StorageBrowserElements = StorageBrowserElements,
 > extends OmitElements<_MessageControl<T>, 'Container'> {
-  ({ variant }: MessageControlProps): React.JSX.Element;
+  ({ variant }: MessageControlProps): React.JSX.Element | null;
 }
+
 export const MessageControl: MessageControl = ({ variant, children }) => {
-  return (
+  const [dismissed, setDismissed] = React.useState<boolean>(false);
```

Comment:
This is one of those local state things that I'm not sure is correct given how we want these to eventually be composable so open to any feedback.